segmentifyandroidsdk/src/main/java/com/segmentify/segmentifyandroidsdk/model/SearchEventResponseModel.kt

```diff
@@ -0,0 +1,6 @@
+package com.segmentify.segmentifyandroidsdk.model
+
+class SearchEventResponseModel {
```

Comment:
why did we create another class for this?

Reply:
because it is different from EventResponseModel, it has response but search response has search instance